weather_dl/download_pipeline/fetcher.py

```diff
@@ -65,6 +65,8 @@ def fetch_data(self, config: t.Dict, *, worker_name: str = 'default') -> None:
             return
 
         client = CLIENTS[self.client_name](config)
+        logger.info(f'By using {self.client_name} datasets, \
+                    users agree to the terms and conditions specified in {client.get_license_url()}')
```

Comment:
This is a great message. However, the location of this log will make it appear on the dataflow job when run remotely. I would prefer that this message appear on the user's local machine before the pipeline is executed:

https://github.com/google/weather-tools/blob/4aa98198ab81d895926d08fb6fd70bc25431d03d/weather_dl/download_pipeline/pipeline.py#L103

 If possible it would be nice it if appeared in a textcolor, or at the warning level.

Reply:
Said changes implemented - placement and log-level changed.